Description: Upstream changes introduced in version 0.5.14-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 cloud-init (0.5.14-0ubuntu1) maverick; urgency=low
 .
   * New upstream release.
    - support for reading metadata and userdata from filesystem
    - support for boot without metadata at all
 .
 The person named in the Author field signed this changelog entry.
Author: Scott Moser <smoser@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- cloud-init-0.5.14.orig/setup.py
+++ cloud-init-0.5.14/setup.py
@@ -23,6 +23,9 @@ from glob import glob
 import os.path
 import subprocess
 
+def is_f(p):
+    return(os.path.isfile(p))
+
 setup(name='cloud-init',
       version='0.5.14',
       description='EC2 initialisation magic',
@@ -40,7 +43,8 @@ setup(name='cloud-init',
                   ('/usr/share/cloud-init', []),
                   ('/usr/lib/cloud-init',
                       ['tools/uncloud-init','tools/write-mime-multipart']),
-                  ('/usr/share/doc/cloud-init', glob('doc/*.txt')),
-                  ('/usr/share/doc/cloud-init/examples', glob('doc/examples/*')),
+                  ('/usr/share/doc/cloud-init', filter(is_f,glob('doc/*'))),
+                  ('/usr/share/doc/cloud-init/examples', filter(is_f,glob('doc/examples/*'))),
+                  ('/usr/share/doc/cloud-init/examples/seed', filter(is_f,glob('doc/examples/seed/*'))),
                   ],
       )
